package.json

```diff
@@ -14,7 +14,7 @@
     "start:mmi": "yarn start --build-type mmi",
     "start:lavamoat": "yarn build:dev dev --apply-lavamoat=true",
     "dist": "yarn build dist",
-    "dist:mv2": "ENABLE_MV3=false yarn build dist",
+    "dist:mv2": "ENABLE_MV3=false yarn build dist  --apply-lavamoat=false --snow=false",
```

Comment:
Could you explain why this would be needed? It is critical that LavaMoat is enabled on all platforms, we cannot disable it for production builds.

Reply:
It is a mistake fix, reverted now. thanks for reviewing 🙏🏻